postgres-appliance/build_scripts/base.sh

```diff
@@ -129,22 +129,41 @@ for version in $DEB_PG_SUPPORTED_VERSIONS; do
 
     # Install 3rd party stuff
 
-    # use subshell to avoid having to cd back (SC2103)
-    (
-        cd timescaledb
-        for v in $TIMESCALEDB; do
-            git checkout "$v"
-            sed -i "s/VERSION 3.11/VERSION 3.10/" CMakeLists.txt
-            if BUILD_FORCE_REMOVE=true ./bootstrap -DREGRESS_CHECKS=OFF -DWARNINGS_AS_ERRORS=OFF \
-                    -DTAP_CHECKS=OFF -DPG_CONFIG="/usr/lib/postgresql/$version/bin/pg_config" \
-                    -DAPACHE_ONLY="$TIMESCALEDB_APACHE_ONLY" -DSEND_TELEMETRY_DEFAULT=NO; then
-                make -C build install
-                strip /usr/lib/postgresql/"$version"/lib/timescaledb*.so
-            fi
-            git reset --hard
-            git clean -f -d
-        done
-    )
+    # Used to avoid installing toolkit if timescaledb wasn't installed for the pgversion.
+    __timescaleDbInstalled="false"
+
+    pushd .
+    cd timescaledb
+    for v in $TIMESCALEDB; do
+        git checkout "$v"
+        sed -i "s/VERSION 3.11/VERSION 3.10/" CMakeLists.txt
+        if BUILD_FORCE_REMOVE=true ./bootstrap -DREGRESS_CHECKS=OFF -DWARNINGS_AS_ERRORS=OFF \
+                -DTAP_CHECKS=OFF -DPG_CONFIG="/usr/lib/postgresql/$version/bin/pg_config" \
+                -DAPACHE_ONLY="$TIMESCALEDB_APACHE_ONLY" -DSEND_TELEMETRY_DEFAULT=NO; then
+            make -C build install
+            strip /usr/lib/postgresql/"$version"/lib/timescaledb*.so
+            __timescaleDbInstalled="true"
+        fi
+        git reset --hard
+        git clean -f -d
+    done
+    popd
```

Comment:
I wanted a check to make sure toolkit isn't installed if timescaledb isn't.
Got rid of the subshell in favour of using pushd/popd to be able to set the variable.

Reply:
regarding this `__timescaleDbInstalled` var...  I had a feeling timescale-toolkit can be used without timescaledb installed (with pure pg). Am I wrong?

Reply:
I'm not sure! Haven't seen in said explicitly either way, guess I just didn't think it'd make much sense.
Want me to remove the check __timescaleDbInstalled?